pkg/detectors/fibery/fibery.go

```diff
@@ -0,0 +1,88 @@
+package fibery
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat    = regexp.MustCompile(detectors.PrefixRegex([]string{"fibery"}) + `\b([0-9a-z.]{44})\b`)
+	domainPat = regexp.MustCompile(detectors.PrefixRegex([]string{"fibery", "domain"}) + `\b([0-9a-z]{5,20})\b`)
+)
+
+// Keywords are used for efficiently pre-filtering chunks.
+// Use identifiers in the secret preferably, or the provider name.
+func (s Scanner) Keywords() []string {
+	return []string{"fibery"}
+}
+
+// FromData will find and optionally verify Fibery secrets in a given set of bytes.
+func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
+	dataStr := string(data)
+
+	matches := keyPat.FindAllStringSubmatch(dataStr, -1)
+	domainMatches := domainPat.FindAllStringSubmatch(dataStr, -1)
+
+	for _, match := range matches {
+		if len(match) != 2 {
+			continue
+		}
+		resMatch := strings.TrimSpace(match[1])
+
+		for _, domainMatch := range domainMatches {
+			if len(domainMatch) != 2 {
+				continue
+			}
+
+			resDomainMatch := strings.TrimSpace(domainMatch[1])
+
+			s1 := detectors.Result{
+				DetectorType: detectorspb.DetectorType_Fibery,
+				Raw:          []byte(resMatch),
+			}
+
+			if verify {
+				timeout := 10 * time.Second
+				client.Timeout = timeout
+				req, err := http.NewRequestWithContext(ctx, "POST", "https://"+resDomainMatch+".fibery.io/api/commands", nil)
```

Comment:
nit: Prefer `fmt.Sprintf` here: `fmt.Sprintf("https://%s.fibery.io/api/commands", resDomainMatch)`

Reply:
Great! Pushed the enhancements. Thanks @ahrav!